src/components/VIcon/VIcon.vue

```diff
@@ -33,7 +35,7 @@ export default {
       /**
        * In `jest` our icons get transformed to Vue components
        */
-      type: process.env.NODE_ENV === 'test' ? Object : String,
+      type: isTest ? Object : String,
```

Comment:
I would like to check that dead code elimination still works when you don't inline the NODE_ENV check. I don't think we have environment based dead code currently but it would be good to add that. We might end up having to revert these checks back to the inline but I'm not sure.

Reply:
I'll make a new issue for this, it seems that this should work because it'll evaluate to 
```js
if (false) {
  // ...
}
```
in the compiled JS.

Reply:
Opened #1030.